src/test/java/cosminsechel/tests/seleniumpart1/Demoqatema.java

```diff
@@ -0,0 +1,32 @@
+package cosminsechel.tests.seleniumpart1;
+
+import cosminsechel.driver.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Demoqatema {
+    public static void main(String[] args) throws InterruptedException {
+        WebDriver driver = BrowserManager.createDriver();
+        driver.get("https://demoqa.com/elements");
+        driver.manage().window().maximize();
+        driver.findElement(By.id("item-0")).click();
+        driver.findElement(By.id("userName")).sendKeys("Cosmin");
+        driver.findElement(By.id("item-1")).click();
+        WebElement checkbox = driver.findElement(By.cssSelector("label[for='tree-node-home']"));
+        checkbox.click();
+        if (checkbox.isSelected()) {
+            System.out.println("checkbox selected");
+
+        }else{
+            System.out.println("checkbox not selected ");
+            closeBrowser();
```

Comment:
Move the closeBrowser(); call after if - else block. We want to close always the browser, not only if we're not selecting the checkbox. Thanks!